src/sempy_labs/_ai.py

```
@@ -367,7 +384,7 @@ def generate_aggs(
                 print(
                     f"{icons.green_dot} '{aggTableName}'[{fromColumn}] -> '{toTable}'[{toColumn}] relationship has been added."
                 )
-            except:
+            except Exception:
```

do you want to swallow the exception here? at least I would surface the exception message as part of the print.

---

I updated the exception message to carry forward.